docs/_edot-sdks/nodejs/migration.md

```diff
@@ -7,8 +7,202 @@ parent: EDOT Node.js
+
+### `apmClientHeaders`
+
+The Elastic [`apmClientHeaders`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#apm-client-headers) option corresponds to the OpenTelemetry [`OTEL_EXPORTER_OTLP_HEADERS`](https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specifying-headers-via-environment-variables) option.
+
+For example: `OTEL_EXPORTER_OTLP_HEADERS=foo=bar,baz=quux`.
+
+### `disableInstrumentations`
+
+The Elastic [`disableInstrumentations`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#apm-client-headers) option corresponds to the OpenTelemetry [`OTEL_NODE_DISABLED_INSTRUMENTATIONS`](https://opentelemetry.io/docs/zero-code/js/configuration/#disable-specific-instrumentations) option. See [Configuration](./configuration) if you want to learn more advanced usage.
+
+For example: `OTEL_NODE_DISABLED_INSTRUMENTATIONS=express,mysql`.
+
+### TODO (double check) `containerId`
+
+The Elastic [`containerId`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#container-id) option corresponds to setting the `container.id` key in [OTEL_RESOURCE_ATTRIBUTES](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_resource_attributes).
+
+For example: `OTEL_RESOURCE_ATTRIBUTES=container.id=my-id`.
+
+### `metricsInterval`
+
+The Elastic [`metricsInterval`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#metrics-interval) option corresponds to the OpenTelemetry [`OTEL_METRIC_EXPORT_INTERVAL`](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader) option.
+
+For example: `OTEL_METRIC_EXPORT_INTERVAL=30000`.
+
+### `cloudProvider`
+
+The Elastic [`cloudProvider`](https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#cloud-provider) option does not corresponds directly to an OpenTelemetry option but you can get similar behaviour by properly setting [`OTEL_NODE_RESOURCE_DETECTORS`](https://opentelemetry.io/docs/zero-code/js/configuration/#sdk-resource-detector-configuration) option. You have to make sure the
+cloud provider is on the list of detectors. Not setting this option is the equivalent of `auto`.
+
+For example: `OTEL_NODE_RESOURCE_DETECTORS=os,env,host,process,gcp`. Will make the agent query for GCP metadata only.
+
+
+
 TODO:
-- migration steps
-- advantages of using EDOT
-- limitations compared to classic agents
-- (optional) performance comparison
\ No newline at end of file
+https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#long-field-max-length
+
+is it maybe attr limits on each signal?
+https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#attribute-limits
+https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#span-limits
+https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#logrecord-limits
```

Comment:
Yes, just the `attribute-limits` link will suffice. Users can use the span or logrecord specific ones if they *really* need to.

It *might* be worthwhile to defer handling this config var to a separate issue to finish documenting all the config vars. The subtlety with attribute limits is that the downstream Elasticsearch might effectively truncate (or at least limit indexing to the first 1000 chars) no matter how the SDK is configured.